src/components/dashboard/Navbar.jsx

```
@@ -17,46 +18,83 @@ const Navbar = ({ handleButtonClick }) => {
-            </svg>
-        </button>
-    </div>
+                        <img src={campushm} alt="Campus" className="w-20 h-auto" />
+                        <h2 className="text-sm font-bold pt-1">Admin Panel</h2>
+                    </div>
+                    <button
+                        onClick={() => setIsOpen(!isOpen)}
+                        className="text-white focus:outline-none z-50 relative"
+                    >
+                        <svg className="w-8 h-8" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                            <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d={isOpen ? "M6 18L18 6M6 6l12 12" : "M4 6h16M4 12h16M4 18h16"} />
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)